db/schema.rb.example

```diff
@@ -11,446 +10,418 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180618184048) do
+ActiveRecord::Schema.define(version: 2018_07_03_164027) do
 
-  create_table "answer_selections", force: true do |t|
+  create_table "answer_selections", force: :cascade do |t|
```

Comment:
Hmm, maybe @Souravirus has thoughts on this and other edits to the schema example file?

Reply:
Yeah, this change looks good to me